app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :posts
+  has_many :comments
+  has_many :likes
+
+  def recent_posts
+    Post.where(user_id: id).order(created_at: :desc).first(3)
+  end
```

Comment:
- [ ] Another way of doing this would be:

```ruby
def recent_post
  posts.order('created_at Desc').limit(3)
end
```

That way, you do it with fewer lines, with a limit, and faster! 💪